src/5.0_to_5.1/calculated-measures/migrateCalculatedMeasureRecord.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  ContentRecord,
+  MdxCompoundIdentifier,
+  MdxFunction,
+  MdxLiteral,
+  parse,
+} from "@activeviam/activeui-sdk-5.0";
+
+/**
+ * Transforms the serialized definition of a calculated measure created with ActiveUI 5.0, into one that is natively supported by ActivePivot.
+ * Mutates `legacyCalculatedMeasureContent`.
+ */
+export const migrateCalculatedMeasureRecord = (
+  legacyCalculatedMeasureContent: ContentRecord,
+  calculatedMeasureName: string,
+): void => {
+  const {
+    expression,
+    properties,
+  }: { expression: string; properties: string[] } = JSON.parse(
+    legacyCalculatedMeasureContent.entry.content,
+  );
+
+  const formatStringExpression = properties
+    ? properties
+        .find((property) => property.startsWith("FORMAT_STRING"))
+        ?.replace("FORMAT_STRING = ", "")
```

Comment:
Is this OK like this, or do you prefer that I user `parse` here, like below ?

Reply:
Now that I am seeing this, I am thinking that indeed `parse` is not the safest, since it relies on the assumption that the expression of a property is necessarily an `MdxLiteral`.

So instead we could just do `const [propertyName, propertyExpression] = propertyString.split(" = ")`.

And we could do the same handling both for `"FORMAT_STRING"` and other prroperties.